owncloudApp/src/main/res/values/strings.xml

```diff
@@ -647,4 +647,8 @@
     <string name="security_enforced_first_option">Passcode lock</string>
     <string name="security_enforced_second_option">Pattern lock</string>
 
+    <!--Deep Links-->
+    <string name="no_account_configured">No account configured previously</string>
+    <string name="no_file_found">No file found</string>
```

Comment:
I would more descriptive strings to advertise to the user the error in a more detailed way. Probably @jesmrec can help us with the error messages.

BTW, I would add the `deep_link` prefix to the string keys